packages/fxa-settings/src/components/PageDeleteAccount/index.test.tsx

```diff
@@ -80,6 +80,26 @@ describe('PageDeleteAccount', () => {
     expect(screen.getByTestId('continue-button')).toBeDisabled();
   });
 
+  it('Requires at least 8 characters to enable Delete button', async () => {
+    renderWithRouter(
+      <AppContext.Provider value={mockAppContext({ account })}>
+        <PageDeleteAccount />
+      </AppContext.Provider>
+    );
+
+    await advanceStep();
+    expect(screen.getByTestId('delete-account-button')).toBeDisabled();
+
+    await act(async () => {
+      await typeByTestIdFn('delete-account-confirm-input-field')('hunter67');
```

Comment:
@jodyheavener Not sure why but maybe because I squashed the commits and pushed again?
Here you can see I'm making use of `typeByTestIdFn()` as well as `hunter67` instead of `hunter6`